packages/legend-application-studio/src/components/editor/editor-group/service-editor/testable/ServiceTestsEditor.tsx

```diff
@@ -158,6 +158,14 @@ export const ExternalFormatParameterEditorModal = observer(
       updateParamValue,
       bindingParamPair,
     } = props;
+    const paramValue = (
+      paramState.varExpression.genericType?.value.rawType === PrimitiveType.BYTE
+        ? atob(
+            (paramState.valueSpec as PrimitiveInstanceValue)
+              .values[0] as string,
+          )
+        : (paramState.valueSpec as PrimitiveInstanceValue).values[0]
+    ) as string;
```

Comment:
why do we need this cast here? we should only cast the result of line 167

Reply:
Fixed thanks!